torch/_inductor/pattern_matcher.py

```diff
@@ -251,14 +251,73 @@ def replace_by_example(
             else contextlib.nullcontext()
         )
 
+        def should_propagate_arg_kwarg_vals(nodes: list[torch.fx.Node]) -> bool:
+            if len(nodes) != 1:
+                return False
+            node = nodes[0]
+            if "arg_kwarg_vals" not in node.meta:
+                return False
+            return node.target in OrderedSet(
```

Comment:
nit: node.target is ...

Reply:
the code looks like this so I can add more things to it in the follow-up PR. I can change this here if you want me to but it's easier to not